api/controllers/controllers.js

```diff
@@ -1,9 +1,18 @@
-const { fetchTopics } = require("../models/models")
+const { fetchTopics, fetchArticleById } = require("../models/models")
 
 exports.getTopics = (req, res, next) => {
     fetchTopics().then((topics) =>{
         res.status(200).send(topics);
     }).catch((err) =>{
         next(err);
     })
+}
+
+exports.getArticleById = (req, res, next) =>{
+    const {article_id} = req.params;
+    fetchArticleById(article_id).then((article)=>{
+        res.status(200).send(article)
```

Comment:
I would send this back as an object on a key of article, then you are future proofing your tests if you wanted to add on more data later. You can always just check the key of article in that case.